code/modules/projectiles/ammunition.dm

```diff
@@ -123,6 +123,7 @@
 	to_chat(user, "<span class='notice'>You empty [src].</span>")
 	for(var/obj/item/ammo_casing/C in stored_ammo)
 		C.forceMove(user.loc)
+		playsound(C, "sound/weapons/casingdrop[rand(1,5)].ogg", 50, 1)

```

Comment:
That might not sound too good. (Especially when emptying the machine gun mag or other large mags)
Maybe add a special sound file for a magdump?

Reply:
Why not?

Reply:
Most games typically use randomized casing sounds. They're also typically quieter than the gunshot by.. A lot. It would probably help.

Reply:
These sounds aren’t both randomized and quieter than the gunshot. What precisely is your point?

Reply:
I'm agreeing with you.